src/controllers/update/update.sign.out.controller.ts

```diff
@@ -2,15 +2,24 @@ import { NextFunction, Request, Response } from "express";
 import { createAndLogErrorRequest, logger } from "../../utils/logger";
 import * as config from "../../config";
 import { isActiveFeature } from "../../utils/feature.flag";
+import { getPreviousPageUrl, isRemoveJourney } from "../../utils/url";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
+  let journey = config.JourneyType.update;
+  let previousPage = `${config.UPDATE_AN_OVERSEAS_ENTITY_URL}${req.query["page"]}`;
+
+  if (isRemoveJourney(req)) {
+    journey = config.JourneyType.remove;
+    previousPage = getPreviousPageUrl(req, config.UPDATE_AN_OVERSEAS_ENTITY_URL);
+  }
```

Comment:
Might be worth a comment explaining why a remove journey needs to call the getPreviousPageUrl function whereas the update journey does not. Or maybe a comment on the function itself.

Reply:
added a comment to say why we use that approach